packages/core/nuxt-theme-module/theme/assets/notifications/cart.ts

```diff
@@ -0,0 +1,22 @@
+import { useUiNotification } from '../../composables';
+
+export const addItem = ({ productName, error, onClick, $i18n }) => {
+  const { send } = useUiNotification();
+
+  if (error) {
+    send({
+      type: 'danger',
+      message: error
+    });
+  } else {
+    send({
+      type: 'success',
+      message: productName ? $i18n.t('Successfully added {PRODUCT_NAME} to the cart', { PRODUCT_NAME: productName }) : $i18n.t('Successfully added product to the cart'),
+      persist: true,
+      action: {
+        text: $i18n.t('Go to Checkout'),
+        onClick
```

Comment:
isn't that always a checkout link here?

Reply:
Now it is. I have moved 
```js
onClick: () => router.push('/checkout/shipping')
```
to inside `addItem` function